pkg/virt-operator/resource/generate/components/secrets.go

```diff
@@ -24,6 +24,7 @@ import (
 // #nosec 101, false positives were caused by variables not holding any secret value.
 const (
 	KubeVirtCASecretName                              = "kubevirt-ca"
+	ExternalKubeVirtCAConfigMapName                   = "external-kubevirt-ca"
```

Comment:
```suggestion
	ExternalKubeVirtCAConfigMapName                   = "kubevirt-external-ca"
```

Reply:
We have `kubevirt-ca` and `kubevirt-export-ca`, so `external-kubevirt-ca` hurts my eyes, but I'll get over it :)